gulpfile.js

```diff
@@ -15,7 +15,7 @@ gulp.task('minor-release', ['bump']);
 gulp.task('major-release', ['major-bump']);
 
 gulp.task('scripts', function() {
-    return gulp.src(paths.js)
+    return gulp.src('./scss/ionic-material-design-lite.scss')
```

Comment:
ups, careful this is the scripts task sory i think i messed you up. this has to be `paths.js` and the sass task `gulp.src(['./scss/ionic-material-design-lite.scss'])`